src/components/NavBar/index.js

```diff
@@ -16,22 +31,30 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-export default function NavBar() {
+const NavBar = (props) => {
+  const { home = [] } = props;
   const classes = useStyles();
 
+  useEffect(() => {
+    props.fetchHomeConfiguration();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
why do you need to disable eslint here?

Reply:
fixed in last commit

Reply:
How have you fixed it?
If you can't fix it, tell me a reason.